great-idea/css/index.css

```diff
@@ -69,76 +69,71 @@ h1, h2, h3, h4, h5 {
 	border: 1px solid black;
 	width: 888px;
 	margin: 0px auto;
-	vertical-align: center;
 	display: flex;
 	flex-direction: column;
-	align: center;
-
+	
 }
 
 	.container-header {
-		border: 4px solid red;
-		/* width: 880px; */
-		height: 250px;
-		margin: 0px 10px 0px 10px;
+		/* border: 4px solid red; */
+		height: 280px;
 		display: flex;
 		flex-direction: column;
 		justify:center;
-		/* z-index: 1; */
-		/* justify-content: space-around; */
 	}
 		
 		.box-nav {
-			border: 1px solid red;
-			text-decoration: none;
-			width: 95%;
-			height: 20px; 
+			/* border: 3px solid yellow; */
+			/* text-decoration: none; */
+			width: 100%;
+			height: 50px; 
 			display: flex;
-			justify-content: flex-start;
+			justify-content: space-between;
 
```

Comment:
You were very close with this nav bar from a flexbox standpoint. It seems you over looked the cross axis. Do you remember how to adjust the cross axis? Reach out if you don't